scvi/model/base/_vaemixin.py

```diff
@@ -126,6 +149,7 @@ def get_latent_representation(
         give_mean: bool = True,
         mc_samples: int = 5000,
         batch_size: Optional[int] = None,
+        _adata=None,
```

Comment:
what is the purpose of this?

Reply:
The problem is that `_inference_loop`, in `_demixin.py` calls `get_marginal_ll` a lot. Unfortunately, this method needs to check the validity of the data at each call, by calling `_validate_anndata`.
`_validate_anndata` takes approximately 1sec to run, which is fine when it is executed once, but this is more of a problem when you call `get_marginal_ll` (or `get_latent_representation`) many times

Reply:
`_adata` bypasses this behavior by assuming that the anndata has already been checked.
One clarification: I added this to `get_latent_representation` because this function can still be called a number of times when you do 1vsAll tests for instance

Reply:
I don't really know if I could do something cleaner, what do you think?

Reply:
Ok so we should consider either a boolean argument (validate_anndata: bool = True) or maybe a private field in the anndata setup dictionary? what do you think @galenxing

Reply:
private field would be like `adata.uns["_scvi"]["_requires_validation"] = False`

Reply:
Went for the latter option!

Reply:
See 1473719